lib/decompress/zstd_decompress_block.c

```diff
@@ -99,6 +100,7 @@ size_t ZSTD_decodeLiteralsBlock(ZSTD_DCtx* dctx,
                 U32 const lhlCode = (istart[0] >> 2) & 3;
                 U32 const lhc = MEM_readLE32(istart);
                 size_t hufSuccess;
+                size_t expectedWriteSize = MIN(ZSTD_BLOCKSIZE_MAX, dstCapacity);
```

Comment:
This is the cause of the last remaining streaming fuzzer error.  It appears in some cases, having the lit buffer stored ZSTD_BLOCKSIZE_MAX beyond dst is enough for it to stomp the extended dictionary in streaming mode, so if a match is at fairly long offset and into this area it can copy incorrectly from the litbuffer.  I had thought that ZSTD_BLOCKSIZE_MAX was guaranteed not to be in the range of where we needed the extDict from previous offsets; still investigating what the correct safe offset should be here.

Reply:
yes, the fuzzer error was fixed by the increased streaming buffer size